### PR TITLE
Run @samccone's memory test on drone

### PR DESCRIPTION
**Pull request by surma**
FYI @crhym3: we are running headless(?) tests on Drone now :)

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ script:
   # ensure the build works
   # this will also run gulp test
   - gulp all
+  # run tests explicitly
```
Comment (crhym3) [inline .drone.yml]: if `gulp all` doesn't run tests anymore, remove `# this will also run gulp test` line above

**Comment by crhym3** [comment]
CI errors out with:

```
Error: Cannot find module '../../node_modules/drool/node_modules/selenium-webdriver'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/var/cache/drone/src/github.com/google/material-design-lite/test/memory/test.js:7:17)
```

Does `drool` install selenium driver by default?

**Comment by surma** [comment]
@crhym3 You mind helping us with this? I feel like the `.drone.yml` is in a sane state right now, but we need chromedriver (and probably chrome?!) on the machine. Not sure what the best way is to set this up.

Also this link from @samccone might be insightful: https://github.com/samccone/drool/blob/master/.travis.yml#L5-L11

**Comment by crhym3** [comment]
Ok. I'll take a look.

**Comment by surma** [comment]
Thank you :)

**Comment by samccone** [comment]
and one more example if you need
https://github.com/twosigma/beaker-notebook/blob/master/.travis.yml#L3-L12

**Comment by addyosmani** [comment]
@crhym3 mind if we assign you as the owner for the reviews on this one? I think you're the current master of the universe when it comes to all things Drone :)

**Comment by crhym3** [comment]
looking at it now. will fix soon.

**Comment by samccone** [comment]
@crhym3 let me know if you get stuck on this

**Comment by samccone** [comment]
:+1: looks like you got it!!!!

**Comment by crhym3** [comment]
Yeah, it should be working now. It's always using chrome latest stable atm.
Feel free to merge if it looks ok.

**Comment by crhym3** [comment]
cla bot and I have agreed to disagree.

**Comment by surma** [comment]
Awesome! Thanks @crhym3! Merging :)

**Pull request merged**